test/io_tests/test_intern_wrapper.py

```diff
@@ -0,0 +1,41 @@
+import os
+import unittest
+from shutil import rmtree
+
+import numpy as np
+
+try:
+    import intern
+except ImportError:
+    intern = None
+
+
+@unittest.skipIf(intern is None, "Needs intern (pip install intern)")
+class TestInternWrapper(unittest.TestCase):
+    def test_can_access_dataset(self):
+        from elf.io.intern_wrapper import InternDataset
+
+        # Choosing a dataset at random to make sure we can access shape and dtype
+        ds = InternDataset("bossdb://witvliet2020/Dataset_1/em")
+        self.assertEqual(ds.shape, (300, 36000, 22000))
+        self.assertEqual(ds.dtype, np.uint8)
+        self.assertEqual(ds.size, 300 * 36000 * 22000)
+        self.assertEqual(ds.ndim, 3)
+
+    def test_can_download_dataset(self):
+        from elf.io.intern_wrapper import InternDataset
+
+        ds = InternDataset("bossdb://witvliet2020/Dataset_1/em")
+        cutout = ds[210:211, 7000:7064, 7000:7064]
+        self.assertEqual(cutout.shape, (1, 64, 64))
```

Comment:
Can you also download directly via the intern API here and check for equality?
(I know that the wrapper is very thin, so this is very unlikely to fail, but I think it's better to be more careful in the tests ;))

Reply:
Definitely! I meant to ask how you wanted me to handle this to keep our tests in this repo isolated from the "in-use" database. My original plan was to download from a dataset and check a handful of voxels for equality, but that feels sloppy. We could also include a small .npy array in the tests directory to check full-array equality of a few cutouts, but that feels like adding unnecessary clutter. 

My instinct is to do the former:

```python
data = InternDataset("bossdb:// ... ") # some known, public dataset
assert data[100, 200, 300] == 137 # magic number
assert data[10, 300, 200] == 42 # magic number
```

How does that sound?

Reply:
> My instinct is to do the former:
> 
> ```python
> data = InternDataset("bossdb:// ... ") # some known, public dataset
> assert data[100, 200, 300] == 137 # magic number
> assert data[10, 300, 200] == 42 # magic number
> ```
> 
> How does that sound?

Yes, I think that's a good solution!

Reply:
(Just added this in the latest commit!)